change/@telekom-design-tokens-ea04da6f-ec3e-4960-86b3-716443e852c7.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix for app bottom shadow",
```

Comment:
I would put some more explicit info in the comment, like `fix: rename shadow/app-bar/bottom/flat to shadow/app-bar/bottom/raised`, including the token names for reference.

Reply:
This is not really true though, we had flat twice, so we still have flat, but now the missing raised is added.

Reply:
aaah, sorry, haha, the new comment is 💯 in any case, thank you very much!